src/logic/safe/store/actions/createTransaction.ts

```diff
@@ -116,7 +116,7 @@ export const createTransaction = (
     sender: from,
     sigs,
   }
-  const safeTxHash = generateSafeTxHash(safeAddress, txArgs)
+  const safeTxHash = await generateSafeTxHash(safeAddress, safeVersion, txArgs)
```

Comment:
Can this throw?

Reply:
It shouldn't, but it could, as we are doing one RPC request to get networkId. Will add it to the try/catch